lib/actions/getVariantsBySku.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+const request        = require('request');
```

Comment:
For the future, can we start using `node-fetch` instead of `request`? It is promise-based so ties in nicely with the rest of the code.